modules/virt-example-nmstate-multiple-interfaces.adoc

```diff
@@ -7,26 +7,53 @@
 
 You can create multiple interfaces in the same node network configuration policy. These interfaces can reference each other, allowing you to build and deploy a network configuration by using a single policy manifest.
 
-The following example snippet creates a bond that is named `bond10` across two NICs and a Linux bridge that is named `br1` that connects to the bond.
+The following example snippet creates a bond that is named `bond10` across two NICs and VLAN that is named `bond10.103` that connects to the bond.
 
 [source,yaml]
 ----
-# ...
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: bond-vlan <1>
+spec:
+  nodeSelector: <2>
+    kubernetes.io/hostname: <node01> <3>
+  desiredState:
     interfaces:
-    - name: bond10
-      description: Bonding eth2 and eth3 for Linux bridge
-      type: bond
-      state: up
+    - name: bond10 <4>
+      description: Bonding eth2 and eth3 <5>
+      type: bond <6>
+      state: up <7>
       link-aggregation:
-        port:
+        mode: balance-rr <8>
+        options:
+         miimon: '140' <9>
```

Comment:
micro-nit, I think we use two spaces for indent. `miimon:` is one space indented.

Reply:
Thanks for catching that!